tests/state_test_util.go

```diff
@@ -274,9 +313,17 @@ func (tx *stTransaction) toMessage(ps stPostState, r params.Rules) (blockchain.M
 		return nil, fmt.Errorf("invalid tx data %q", dataHex)
 	}
 
-	intrinsicGas, err := types.IntrinsicGas(data, nil, to == nil, r)
-	if err != nil {
-		return nil, err
+	var intrinsicGas uint64
+	if isTestExecutionSpecState {
+		intrinsicGas, err = useEthIntrinsicGas(data, to == nil, r)
+		if err != nil {
+			return nil, err
+		}
+	} else {
+		intrinsicGas, err = types.IntrinsicGas(data, nil, to == nil, r)
+		if err != nil {
+			return nil, err
+		}
```

Comment:
```suggestion
	var intrinsicGas uint64
	if isTestExecutionSpecState {
		intrinsicGas, err = useEthIntrinsicGas(data, to == nil, r)
	} else {
		intrinsicGas, err = types.IntrinsicGas(data, nil, to == nil, r)
         }
         if err != nil {
		return nil, err
	}
```

Reply:
As-Is:
if ether's gas calc fails:
  set error to `err` and return
if kaia's gas calc fails:
  set error to `err` and return

I thought that this is equiv with the suggested change.
if ether's gas calc fails:
  set error to `err`
if kaia's gas calc fails:
  set error to `err`
if err is not nil return

Thus, I think there is no missing information with this change. Please teach me if there's another point .

Reply:
It was my mistake. This applies.
commit: https://github.com/kaiachain/kaia/pull/129/commits/11899f7495a691917fbc01e031a213e6ef88a205